roles/ceph-mon/tasks/docker/main.yml

```diff
@@ -88,3 +88,6 @@
     - groups[restapi_group_name] is defined
     - inventory_hostname == groups.mons|last
     - not mon_containerized_deployment_with_kv
+
+- include: openstack_config.yml
```

Comment:
We need a docker version for this. `ceph` command is not available on the host so we have to `docker exec <mon> <command>`. No need for a specific host to run that on since the Docker Ceph monitor bootstrap happens serially.